src/main/java/seedu/manager/event/EventList.java

```diff
@@ -67,4 +84,19 @@ public boolean removeEvent(String eventName) {
         }
         return false; // Event not found
     }
+
+    /**
+     * Adds a participant to an existing event.
+     *
+     * @param participantName the name of the participant to be added.
+     * @param eventName the name of the event to which the participant will be added.
+     */
+    public void addParticipantToEvent(String participantName, String eventName) {
+        for (Event event : eventList) {
+            if (event.getEventName().equals(eventName)) {
+                event.addParticipant(participantName);
+                break;
```

Comment:
I think this "break" statement could be changed to "return" to make it clear that the method is being exited from?